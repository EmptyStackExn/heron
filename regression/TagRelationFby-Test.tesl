time relation h1 = [1. 2. ] -> h2
time relation h2 = 5.0
@step
@step
@step
@step
@output tikz select h2 h1