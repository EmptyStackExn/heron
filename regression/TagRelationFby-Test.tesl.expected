  \node[date] at (1, 0)  {5.} ;
  \node[date] at (1, -1)  {1.} ;
  \node[date] at (2, 0)  {5.} ;
  \node[date] at (2, -1)  {2.} ;
  \node[date] at (3, 0)  {5.} ;
  \node[date] at (3, -1)  {5.} ;
  \node[date] at (4, 0)  {5.} ;
  \node[date] at (4, -1)  {5.} ;
